src/Behat/Behat/Tester/Cli/RerunController.php

```diff
@@ -74,6 +74,9 @@ public function configure(Command $command)
         $command->addOption('--rerun', null, InputOption::VALUE_NONE,
             'Re-run scenarios that failed during last execution.'
         );
+        $command->addOption('--rerun-only', null, InputOption::VALUE_NONE,
+            'Re-run scenarios that failed during last execution or exit if no file is found.'
```

Comment:
I think this is clearer than "no file is found" (which is an implementation detail):
```suggestion
            'Re-run scenarios that failed during last execution, or exit if there were no failures.'
```